data/mods/My_Sweet_Cataclysm/sweet_comestibles.json

```diff
@@ -62,7 +62,7 @@
     "name": "graham cracker",
     "symbol": "%",
     "container": "null",
-    "description": "Dry and sugary, these crackers will leave you thirsty, but go good with some chocolate and marshmallows.  They shiver under your touch.  Weird!",
+    "description": "These dry and sugary crackers are thirst-provoking and delicious with marshmallows and chocolate.  You can feel them shiver occasionally when you hold them.  Weird!",
```

Comment:
The first sentence must stay the same as the one in vanilla.

Reply:
Thiniking about it a bit more, I'm not convinced the rewrite of the second sentence is necessary. It just makes it longer which not really what we want since we want the difference from vanilla to be discrete.